packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -38,6 +38,8 @@ import {
   Trash,
   MessageCircle,
 } from "@calcom/ui/components/icon";
+import { AddMembersWithSwitch } from "@calcom/web/components/eventtype/EventTeamTab";
+import type { Host } from "@calcom/web/pages/event-types/[type]";
```

Comment:
These imports are causing the issues, when I removed them type-check passed

Reply:
Yeah, its happening when we import something outside of packages folder i guess, i tried moving the type to a different folder inside packages and it didn't throw any error. Not sure whether refactoring is the right way or there is some other easy alternatives

Reply:
I am also not quite sure. @zomars is there another way to reuse these components/types here in `packages/app-store`?

Reply:
They will need to be extracted to a shared file in one of our packages/ files - we can't import web into any internal packages :)

Reply:
Damn, that's going to be a lot of efforts i guess, anyways thanks @sean-brydon, appreciate it :)

Reply:
Extracting didn't help, it still throws an error in a file which i didn't modify :(